CMakeLists.txt

```diff
@@ -74,11 +74,6 @@ if (XSD_FOUND)
                               ${PROJECT_SOURCE_DIR}/src/schema
                       COMMENT "Copying new XSD wrappers to source tree"
                       VERBATIM)
-else (XSD_FOUND)
-    # Use pre-generated sources & CodeSynthesis headers shipped with the project
```

Comment:
Is there a reason to take this out? The readme still suggests having xsd installed is optional.

Reply:
That's primarily so that a system copy of XSD is always used: which will receive updates from the repositories etc. Our bundled copy is a snapshot of the XSD code that will not be updated unless we remember to update it manually whenever there's a new upstream release. 

I'll update the readme to reflect this.

Reply:
Ah, hold on. I messed this up. I didn't need to remove the whole section. Fixing now.

Reply:
I've updated the CMakeLists.txt file and the Readme too. XSD is now a compulsory requirement, and a new option `REGENERATE_BINDINGS` can be called to generate the bindings. 

I've also made some tweaks to the doc generation bit.